src/components/__tests__/InfiniteHits.js

```diff
@@ -118,6 +154,27 @@ it('renders correctly on the last page', () => {
   expect(wrapper.html()).toMatchSnapshot();
 });
 
+it('expect to call showPrevious on click', () => {
+  const showPrevious = jest.fn();
+
+  __setState({
+    ...defaultState,
+    widgetParams: {
+      ...defaultState.widgetParams,
+      showPrevious: true,
+    },
+    showPrevious,
+  });
+
+  const wrapper = mount(InfiniteHits);
+
+  expect(showPrevious).not.toHaveBeenCalled();
```

Comment:
```suggestion
  expect(showPrevious).toHaveBeenCalledTimes(0);
```

For better error messages and easier refactor.